manifests/init.pp

```diff
@@ -99,6 +99,9 @@
   Optional[Stdlib::Absolutepath] $pulp_ca_cert = $foreman_proxy_content::params::pulp_ca_cert,
   Integer[0] $pulp_worker_timeout = $foreman_proxy_content::params::pulp_worker_timeout,
 
+  Boolean $pulp3 = $foreman_proxy_content::params::pulp3,
+  Boolean $pulp_master = $foreman_proxy_content::params::pulp_master,
```

Comment:
This is already defined -- https://github.com/theforeman/puppet-foreman_proxy_content/pull/222/files#diff-60ae41fd0a31977447947f59940ee9a4R142